src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+enum RenderStatus {
+  Loading = 'Loading',
+  Error = 'Error',
+  ShowPeople = 'ShowPeople',
+}
```

Comment:
you can change names a little to follow promise statuses
```suggestion
enum RenderStatus {
  PENDING = 'pending',
  REJECT = 'reject',
  RESOLVE = 'resolve',
}
```